### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by yatil** [w3c/developer-tools#35]
Hi, there are several accessibility issues with the page: 
- first and foremost contrasts. To get barely enough contrast with grey text on a white background, the text needs to be at least #777, currently it is #747C83 which is way too light. This is easy to check with Firefox [WCAG Contrast Checker add-on](https://addons.mozilla.org/en-us/firefox/addon/wcag-contrast-checker/). Of course the green on the top is too light with white text [as well](http://leaverou.github.io/contrast-ratio/#white-on-%2375C9B3), try #498172 instead, or: [black text](http://leaverou.github.io/contrast-ratio/#black-on-%2375C9B3). 
- The alternative text of the W3C/DEV logo on the bottom should be “W3C Developers”, not logo-w3c-developers.
- The alternative text of the W3C logo on the bottom should be “W3C”, not w3c-logo.
- The octocat should have a null alt, as it is mainly decorative (that means `alt=""`).
- All the other logos and images should have different alternative texts. (There is a [W3C tutorial on that issue](http://www.w3.org/WAI/tutorials/images/)).
- Main navigation items throw the user out of context without any indication.
- Focus indication is weak.
- Donation form barely accessible.
- Above the tools there should be another H1 to indicate to users that this is the main heading, else they might jump directly to the Github heading as it is the only H1 on the page.
- The `section id="tools"` should probably be a `main` element (or the `main` element should wrap `#tools`, `#contribute` and `#donate` sections).

**Comment by yatil** [comment]
BTW: To avoid accessibility issues like that, you can check your page using [Easy Checks](http://www.w3.org/WAI/eval/preliminary).

**Comment by tripu** [comment]
Thank you for such detailed description, @yatil!

I'll try to help @guibbs to fix some of those.

**Comment by yatil** [comment]
👍

**Comment by gbaudusseau** [comment]
Hi @yatil ! Thank you for this feedback! @tripu I will have to fix #32 first.Thank you both @yatil @tripu !

**Comment by shawna-slh** [comment]
Hi y'all. This is becoming a significant issue, as we are not able to point people to this site because of the accessibility errors.

**Comment by dontcallmedom** [comment]
terribly sorry, I'm only noticing this issue now. will try to get fixed asap

**Comment by dontcallmedom** [comment]
having looked into it:
* it looks like all the `alt` bugs have been fixed
* the contrast issues seem to have been fixed
* the `<main>` issue has been fixed
* the `<h1>` issue has been fixed

This leaves (potentially):
* [ ]  Main navigation items throw the user out of context without any indication.
* [ ]   Focus indication is weak.
* [ ]   Donation form barely accessible.

I'm not sure about the "weak" focus indication - I see the usual outline when keyboard navigating, but I may be missing something.

Likewise, I'm not sure how to interpret the "main navigation" report.

Regarding the donation form, is this about https://www.w3.org/support/donate/?recipient=w3c ? (for sake of clarity)?

**Comment by shawna-slh** [comment]
Thanks so much, Dom, for looking into this right away!

(Eric might not be able to respond so quickly.)

I looked at the contrasts.
Ideally, W3C sites meet WCAG Level AAA, which would be:
* 7:1 for most text
* 4.5:1 for large text
At a minimum, W3C sites should meet AA, which would be:
* 4.5:1 for most text
* 3.1 for large text

From a quick check, here's what I'm getting for some of the contrast:
Main page:
* Learn orange 3.4:1
* Tools blue 3.9:1
* Get Involved 3.5:1
* web-platform-tests 3.6:1
* Explore button 4.2:1
* W3C logo at the bottom is 4.0:1

https://www.w3.org/support/:
* main gray text on white: 4.6:1 ?
* Learn more button 4.8:1
* first donate button 4.2:1
* second donate button 3.3:1

I didn't check to see if the buttons would be considered "large text".

In any case, it would be nice if the contrasts were stronger, so as to improve readability for many people, and not call into question whether or not the pages meets WCAG.

**Comment by dontcallmedom** [comment]
thanks will get those fixed asap

**Comment by dontcallmedom** [comment]
I've taken a stab at it - can you check @slhenry ?

**Comment by shawna-slh** [comment]
Contrasts on https://www.w3.org/developers/ much better!
(https://www.w3.org/support/: could still us improvement)
I didn't do a thorough check of everything.

/me now points to this site as an example. :-)
